packages/core/src/index.ts

```diff
@@ -95,18 +99,19 @@ export const initialize = async ({
   tracking.enable({
     allRejections: true,
     onUnhandled: (_: any, error: Error) => {
-      let message = `Unhandled promise rejection: ${error}`;
-      let st = "";
-      if (error instanceof Error) {
-        message = unhandledPromiseRejectionPrefix + error.message;
-        st = error.stack || "";
-      }
-      return NativeModules.EmbraceManager.logMessageWithSeverityAndProperties(
-        message,
-        ERROR,
-        {},
-        st,
-      );
+      // TODO REFACTOR WHEN iOS IMPLEMENT THE METHOD
```

Comment:
can this be uncommented now that we have `logMessageWithSeverityAndProperties` implemented on the Swift side?

Reply:
I am working in other branch, let it commented by now